CHANGES.md

```diff
@@ -23,10 +23,10 @@ and workloads.
 * Controller
   * Instrumented the proxy-injector to provide additional metrics about
     injection (thanks @Pothulapati!)
-  * Added support for `config.linkerd.io/skip-webhook` annotation on namespaces
-    to prevent control plane components from being installed to specific
-    namespaces; this fixes situations in HA deployments where the proxy-injector
-    is installed in `kube-system` (thanks @hasheddan!)
+  * Added support for `config.linkerd.io/skip-webhooks: disabled` label on
+    namespaces so that the pods creation events in these namespaces are ignored
```

Comment:
should `pods` be `pod's` or `pod` here?

also, is `by the proxy injector. namespaces;` on line 28 a typo?

Reply:
I can fix this typo in the email, but at this point I'm going to let the build finish so we can get this cut today!